tests/dbus_service_mocks/network_connection_settings.py

```diff
@@ -1,51 +1,31 @@
 """Mock of Network Manager Connection Settings service."""
 
-from dbus_fast import Variant
+from copy import deepcopy
+from ipaddress import IPv4Address, IPv6Address
+import socket
+
+from dbus_fast import DBusError, Variant
 from dbus_fast.service import PropertyAccess, dbus_property, signal
 
 from .base import DBusServiceMock, dbus_method
 
 BUS_NAME = "org.freedesktop.NetworkManager"
 DEFAULT_OBJECT_PATH = "/org/freedesktop/NetworkManager/Settings/1"
 
-SETTINGS_FIXTURE: dict[str, dict[str, Variant]] = {
-    "connection": {
-        "id": Variant("s", "Wired connection 1"),
-        "interface-name": Variant("s", "eth0"),
-        "llmnr": Variant("i", 2),
-        "mdns": Variant("i", 2),
-        "permissions": Variant("as", []),
-        "timestamp": Variant("t", 1598125548),
-        "type": Variant("s", "802-3-ethernet"),
-        "uuid": Variant("s", "0c23631e-2118-355c-bbb0-8943229cb0d6"),
-    },
+# NetworkManager Connection settings skeleton which gets generated automatically
+# Created with 1.42.4, using:
+# nmcli con add type ethernet con-name "Test"
+# busctl call org.freedesktop.NetworkManager /org/freedesktop/NetworkManager/Settings/5 org.freedesktop.NetworkManager.Settings.Connection GetSettings --json=pretty
```

Comment:
I did not know `busctl` could return you output as prettified json. You definitely just saved me a lot of future time jsonifying output from `gdbus` with this 😄